services/services.go

```diff
@@ -258,3 +264,75 @@ func (i PlacementInstances) String() string {
 	strs = append(strs, "]")
 	return strings.Join(strs, "")
 }
+
+// NewElectionOptions returns an empty ElectionOptions.
+func NewElectionOptions() (ElectionOptions, error) {
+	eo := electionOpts{
+		leaderTimeout: defaultLeaderTimeout,
+		resignTimeout: defaultResignTimeout,
+	}
+
+	h, err := os.Hostname()
+	if err != nil {
+		return nil, err
+	}
+	eo.hostname = h
+
+	return eo, nil
+}
+
+type electionOpts struct {
+	leaderTimeout time.Duration
+	resignTimeout time.Duration
+	ttl           int
+	hostname      string
```

Comment:
I think that conflicts too much with the `LeaderValue` in `CampaignOptions`, and that they should remain separate since these options are service-wide and campaign opts are per-campaign